docs/_documentation/intro.md

```diff
@@ -31,7 +31,7 @@ The goal of Nakadi (**ნაკადი** means "stream" in Georgian) is to pro
 ### Links
 
 Read more to understand *The big picture* 
-[Architecture for data integration](https://pages.github.bus.zalan.do/core-platform/docs/architecture/data_integration.html) 
+[Architecture for data integration](https://zalando.github.io/nakadi/manual.html#data_integration_at_zalando) 
 
 Watch the talk [Data Integration in the World of Microservices](https://clusterhq.com/2016/05/20/microservices-zalando/) 
```

Comment:
Unfortunately this link also gone :( 
Please change it to https://www.youtube.com/watch?v=SbVQBHRAFXA

Reply:
It's not gone, it's where the new page should be :)

Reply:
Didn't get the joke :confused: Please just change the link to the video :pray: .

Reply:
It's not a joke, it is the location of the new page generated by the previous file. But I can add the video as well.

Reply:
dfc0836